app/saphyr/saphyr.rb

```diff
@@ -6,6 +6,10 @@ def self.table_name_prefix
     'saphyr_'
   end
 
+  def self.library_attributes
+    []
```

Comment:
Same comments as ONT. This and library factory should probably raise an unsupported error.